screenpy/actor.py

```diff
@@ -165,43 +165,43 @@ def attempts_to(self: SelfActor, *actions: Performable) -> None:
             self.perform(action)
 
     def was_able_to(self: SelfActor, *actions: Performable) -> None:
-        """Alias for :meth:`~screenpy.actor.Actor.attempts_to`, for test setup."""
+        """Alias for :meth:`~screenpy.actor.Actor.attempts_to`."""
```

Comment:
I also thought i'd take this prescriptive "you should use it here" out. Folks can use these wherever they want!